### PR TITLE
Updates for SloanDigitalSkySurvey v0.0.4

### PR DESCRIPTION
**Pull request by kbarbary**
Updates for rgiordan/SloanDigitalSkySurvey.jl#4. This PR won't pass until that one is merged and a new version tagged.

### CODE REVIEW

**Comment by jeff-regier** [comment]
Hi Kyle, Would you rerun Travis (make a trivial change, I guess) now that the SloanDigitalSkySurvey.jl pr is approved, once you've tagged a new version of SloanDigitalSkySurvey.jl?

If any further breaking changes to SloanDigitalSkySurvey.jl are required, please copy that package over to Celeste.jl rather than continuing to use the standalone version of it...it's not a great workflow to have to make PRs in lock step like this, though this works for now.

**Comment by kbarbary** [comment]
OK, I'll tag a version there. Travis can be rerun through their web API without a new commit; will do that also.

I agree that this workflow is suboptimal. I don't foresee any immediate further changes to SloanDigitalSkySurvey.jl though, so we should be OK for the time being.

File: test/runtests.jl [hunk 1]
```diff
@@ -8,11 +8,28 @@ using SampleData
...
+                 "test_elbo_values.jl",
+                 "test_images.jl",
+                 "test_kl.jl",
+                 "test_misc.jl",
+                 "test_optimization.jl",
+                 "test_sensitive_float.jl",
+                 "test_transforms.jl"]
+end
+
+for testfile in testfiles
```
Comment (jeff-regier) [inline test/runtests.jl:26]: Nice idea!

File: test/test_elbo_values.jl [hunk 2]
```diff
@@ -422,7 +421,7 @@ function test_trim_source_tiles()
       fill(1.0, length(loc_ids)), 0.06)
     @test_approx_eq_eps(
       elbo_full.d[non_loc_ids, 1] ./ elbo_trim.d[non_loc_ids, 1],
-      fill(1.0, length(non_loc_ids)), 2e-3)
+      fill(1.0, length(non_loc_ids)), 4e-3)
```
Comment (kbarbary) [inline test/test_elbo_values.jl:424]: Quick note: I had to bump the tolerance here to get tests to pass locally. I assumed the 2e-3 wasn't particularly well-motivated and that 4e-3 is still OK. Not sure how this bit of code could have been affected by these WCS-related changes.

File: test/runtests.jl (hunk 1 above)
Comment (kbarbary) [inline test/runtests.jl:26]: Stole it from the DataFrames.jl tests. :)

File: test/test_elbo_values.jl (hunk 2 above)
Comment (jeff-regier) [inline test/test_elbo_values.jl:424]: That should be fine

**Pull request merged**